k-distribution/include/kframework/builtin/domains.md

```diff
@@ -522,7 +522,7 @@ number of unbound keys being mached. In other words, one unbound variable is
 linear, two is quadratic, three is cubic, etc.
 
 ```k
-  syntax Set ::= Set Set                  [left, function, functional, hook(SET.concat), klabel(_Set_), symbol, assoc, comm, unit(.Set), idem, element(SetItem), format(%1%n%2)]
+  syntax Set ::= Set Set                  [left, function, hook(SET.concat), klabel(_Set_), symbol, assoc, comm, unit(.Set), idem, element(SetItem), format(%1%n%2)]
```

Comment:
@dwightguth this seems sound to me (ok to remove the extra axiom), but it also seems incorrect. `_Set_` should be functional too, right?

Reply:
No, this is a correct change. I marked it functional originally because the assumption was that sets are idempotent, but we've been treating sets as nilpotent instead, so `SetItem(X) SetItem(X)` is bottom, which means this can't be functional. `unionSet` ought to be functional though.